test/minitest/test_minitest_mock.rb

```diff
@@ -227,17 +227,13 @@ def test_same_method_expects_are_verified_when_all_called
   end
 
   def test_same_method_expects_blow_up_when_not_all_called
-    mock = Minitest::Mock.new
-    mock.expect :foo, nil, [:bar]
-    mock.expect :foo, nil, [:baz]
-
-    mock.foo :bar
-
-    e = assert_raises(MockExpectationError) { mock.verify }
+    @mock = Minitest::Mock.new
+    @mock.expect :foo, nil, [:bar]
+    @mock.expect :foo, nil, [:baz]
```

Comment:
looking at the whole file now, this change makes more sense. I'm starting to wonder if the setup method shouldn't be unfactored as it really isn't saving much of anything. I just have a kneejerk reaction to `@`-only changes.